builder-20/eol-buildpack/buildpack.toml

```diff
@@ -0,0 +1,8 @@
+api = "0.9"
+
+[buildpack]
+  id = "heroku/eol-warning"
+  version = "1.0.0"
+
+[[stacks]]
+  id = "*"
```

Comment:
With Buildpack API 0.10, `[[stacks]]` is no longer required (it was only temporarily required due to the `pack buildpack package` bug which is now fixed).

```suggestion
```